recipes/eigen/run_test.bat

```diff
@@ -0,0 +1,3 @@
+C:\MinGW\bin\g++ -I%LIBRARY_INC% -o test test.cc
```

Comment:
Question: Here we are using AppVeyor's MinGW, but there is also a mingw package in the default channel. Which one should we use/recommend? Or it does not matter?